### PR TITLE
[WIP] Lecture on Distributed Hash Tables

### PR DESCRIPTION
**Pull request by glouppe**
Let's discuss this lecture in this thread.

### CODE REVIEW

File: assets/style.css [hunk 1]
```diff
@@ -65,6 +69,10 @@ a:hover {
     text-decoration: underline;
 }
 
+img {
+    width: 100%;
+}
+
```
Comment (glouppe) [inline assets/style.css]: Please dont make changes on the templates like these, since it will affect all other lectures.

**Comment by glouppe** [comment]
Thanks a lot Joeri! This look good at first sight, but I'll need to go over the slides in more details. I will do that later this week.

Would it be possible to squash and rebase your branch so that the history looks good?

As much as possible, I would try not to mix with changes on the template style, or propose those changes in a separate PR please :)

**Comment by JoeriHermans** [comment]
I'll do the style requests in separate PR's from now :) Sorry

Is it also ok if I rebase the Git history before we merge the branch to master?

**Comment by JoeriHermans** [comment]
Ping @glouppe  http://joerihermans.com/~joeri/lsdb/?p=lecture_x_dht.md (has unmerged changes from master, that's why theme is missing).

Joeri

**Comment by glouppe** [comment]
I'll make a more detailed review later, but just wanted to say that this looks fantastic! Thanks a lot for this :)

**Comment by glouppe** [comment]
Merging this for now, I'll review later.

**Pull request merged**